src/main/resources/Chunks.json

```diff
@@ -1,15 +1,29 @@
-{
-  "Chunk":{
-    "Chunk Number": 1,
-    "Chunk Name": "",
-    "Mobs": [
-      {"Name": "", "Level":1}
-      ],
-    "Skilling Resources":[],
-    "Item Spawns": [""],
-    "Processing Tools": [""],
-    "Shops":[],
-    "Other Notes":""
-  },
-  
-}
+[
+  {
+    "chunkNumber": 1,
+    "chunkName": "1",
+    "mobs": [
+      {
+        "name": "",
+        "combatLvl": 2,
+        "drops":[""]
```

Comment:
No need to store drops in json file, just adds more work/duplication. Simply lookup the mobs database to get drops.

Reply:
Solved